www/class/centreonAuth.LDAP.class.php

```diff
@@ -341,16 +323,23 @@ public function updateUserDn()
                     }
                     $query = "SELECT cg_id FROM contactgroup WHERE cg_name IN (" . $listGroupStr . ")";
                     $res = $this->pearDB->query($query);
-                    /*
-                     * Insert the relation between contact and contact group
-                     */
-                    while ($row = $res->fetchRow()) {
+
+                    // Insert the relation between contact and contactgroups
+                    while ($row = $res->fetch()) {
                         $query = "INSERT INTO contactgroup_contact_relation
     	            						(contactgroup_cg_id, contact_contact_id)
-    	            					VALUES (" . $row['cg_id'] . ", " . $contact_id . ")";
-                        $this->pearDB->query($query);
+    	            					VALUES (:ldapCg, :contactId)";
+                        $stmt = $this->pearDB->prepare($query);
```

Comment:
The statement does not need to be prepared for each row
So it should be prepared just once before the while loop